pkgs/tools/misc/pandoc-drawio-filter/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, drawio
+, pandocfilters
+, xvfb-run
+, lib
+}:
+
+let
+  tag = "1.0";
+in
+
+buildPythonApplication {
+  name = "pandoc-drawio-filter";
+  version = tag;
+
+  src = fetchFromGitHub {
+    owner = "tfc";
+    repo = "pandoc-drawio-filter";
+    rev = tag;
+    sha256 = "sha256-sVjE8fCV7XLrnSVYu1RYU+QflcCNrhLPulE0AQQQWt4=";
+  };
+
+  propagatedBuildInputs = [
+    drawio
+    pandocfilters
+    xvfb-run
```

Comment:
I don't think `xvfb-run` needs to be here, but I may be wrong.

To my understanding, `buildInputs` is not for build _tools_ that need to run during the build. We should use `nativeBuildInputs` for that purpose.

Reply:
xvfb is not a build tool in this context. the python script runs drawio through xvfb because drawio does not properly work without an x server.

Reply:
I see. It seems that this would be effective with invocations from within another derivation, but not outside of one, e.g. when installed, as the stdenv setup script doesn't run there.
With a wrapper, you could make it a "proper" runtime dependency.

Reply:
you're right, i am going to wrap it. it does make sense to have the python script not know about this.